uberduck_ml_dev/exec/parse_data.py

```diff
@@ -0,0 +1,58 @@
+# AUTOGENERATED! DO NOT EDIT! File to edit: nbs/exec.parse_data.ipynb (unless otherwise specified).
+
+__all__ = ['CACHE_LOCATION', 'STANDARD_MULTISPEAKER', 'STANDARD_SINGLESPEAKER', 'FORMATS']
+
+# Cell
+
+import argparse
+import os
+from pathlib import Path
+
+import sqlite3
+from tqdm import tqdm
+
+from ..data.cache import ensure_speaker_table
+from ..data.parse import _log_filelists, _write_db_to_csv
+
+CACHE_LOCATION = Path.home() / Path(".cache/uberduck/uberduck-ml-exp.db")
+STANDARD_MULTISPEAKER = "standard-multispeaker"
+STANDARD_SINGLESPEAKER = "standard-singlespeaker"
```

Comment:
Seems like we define these variables in multiple places as well. Let's define them in 1 place.